File: pset1/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // left side
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height; j++)
        {
            if (j + 1 < height - i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        // 2 spaces
        printf("  ");

        // right side
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}

File: pset6/mario-more/mario.py
from cs50 import get_int

# Input pyramid height (1 ~ 8)
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Create 2 pyramids
for h in range(height):
    # Create left pyramid
    print(" " * (height - h - 1) + "#" * (h + 1), end="")
    # Insert 2 spaces
    print(" " * 2, end="")
    # Create right pyramid
    print("#" * (h + 1))
